analytics-sidecar-vertx-client-shaded/build.gradle

```diff
@@ -47,7 +47,7 @@ configurations {
 
 dependencies {
     shadow(group: 'org.slf4j', name: 'slf4j-api', version: "${project.slf4jApiVersion}")
-    api(project(':analytics-sidecar-vertx-client'))
+    implementation(project(':analytics-sidecar-vertx-client'))
```

Comment:
Unrelated; please revert. If it is a problem, we should fix it in another patch.